_examples/structs/test.py

```diff
@@ -53,3 +53,18 @@
 except Exception as err:
     print("caught error: %s" % (err,))
     pass
+
+class S2_descendant(structs.S2):
```

Comment:
what about `s/S2_descendant/S2Kid/` ?

Reply:
I renamed it to S2Child, so that it is more in line with standard terminology.